lesson01/assignment/scripts.js

```diff
@@ -30,4 +34,96 @@ document. addEventListener("DOMContentLoaded", function(){
     inputField.value = ''
   })
 
-})
\ No newline at end of file
+  form1.addEventListener('submit', function(event){
+    // Prevent the form from trying to submit to a server
+    event.preventDefault()
+
+    // An array that we can transform based on user input
+    let data1 = ['apples', 'bananas', 'carrots', 'bread']
+
+    const inputField1 = document.querySelector('#form1 .input')
+    const userItem = inputField1.value
+
+
+    // If the user's input was a word, push it into the array and continue
+    if(isNaN(userItem)){
```

Comment:
This will only be NaN if we try to turn it into an integer with `parseInt(userItem)`. Did this code work in the browser in testing?